src/events/cd-event-ticket.vue

```diff
@@ -90,6 +96,14 @@
 </script>
 <style scoped lang="less">
   @import "~@coderdojo/cd-common/common/_colors";
+  .before-disabled {
+    &:before {
+      background-image: radial-gradient(circle at .2% 25%, @cd-white .4em, transparent .5em), radial-gradient(circle at .2% 25%, #a9a9a9 .6em,transparent .7em) !important;
```

Comment:
Avoid `!important` at all costs.

In this scenario, it's as simple as moving this CSS to the bottom of the `<style>` tag. CSS cascades, so it will always take the last styles that are defined. Specificity matters too, but shouldn't here. Adapt your LESS to be like I said above, but make sure all your disabled styles are at the bottom.